Sources/Classes/core/Protocols/MiniAppBridgeDelegate.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+/**
+ Public Protocol that will be used by the Mini App to communicate
+ with the Native implementation for sending string data through interface 1 mechanism
+ */
+public protocol MiniAppBridgeDelegate: AnyObject {
```

Comment:
`MiniAppBridgeDelegate` is too generic. Please change to UniversalBridgeDelegate or you can move inside `MiniAppMessageDelegate`

Reply:
I have removed the implementation of `SendJsonToMiniApp()` from the `MiniApp.swift`.